lib/Transforms/Instrumentation/ShadowMemSeaDsa.cc

```diff
@@ -555,10 +579,15 @@ bool ShadowDsaImpl::runOnFunction(Function &F) {
   // -- infer types for PHINodes
   inferTypeOfPHINodes(F);
 
+  solveLoads(F);
```

Comment:
Please add an option for this. It is convenient for debugging to disable this during testing.
It is ok if `solveLoads` is ran by default.